docs/index.md

```diff
@@ -2276,6 +2277,12 @@ certain functions of the mmj2 tool:
   must be used as the justification. You can do the same by
   selecting it as the "Label".
 
+#### Unification
+
+Metamath-lamp's current unification algoritm is only unidirectional, that is, it can only substitute in one direction to find a match. In the technical literature this "unification" algorithm is often called "matching" instead of being considered full syntactic unification. Thiz is contrast to ither tools, like mmj2, which implement full syntactic unification.
+
```

Comment:
typo `Thiz is contrast to ither tools` -> `This is in contrast to other tools`

Reply:
Agreed. I made these edits on my smart phone, and clearly there are drawbacks to doing things that way :-).